features/common/BUILD.bazel

```diff
@@ -364,11 +375,15 @@ feature_set(
         ":exceptions",
         ":use_lld",
         ":lto",
-        ":minsize",
         ":symbol_garbage_collection",
         ":dbg",
         ":fastbuild",
         ":opt",
+
+        # To overwrite the level to minsize with feature flags, it needs to be
+        # listed after the compilation modes.
+        ":minsize",
```

Comment:
Based on the flag names in the feature definition (`-mllvm`), it seems this this may be an LLVM-only feature and this might be an error for GCC-based compilers.

I'd suggest adding this to the `feature_set` for toolchains where we know the compiler is LLVM.  E.g., `//platforms/riscv32/features:rv32imcb`.

Reply:
Thanks, I moved the llvm flags to rv32 features.